site/en/setup.md

```diff
@@ -11,13 +11,24 @@ OCR-D's software is a modular collection of many projects (called _modules_)
 with many tools per module (called _processors_) that you can combine freely
 to achieve the workflow best suited for OCRing your content.
 
+## System requirements
+
+Minimum system requirements
+* 8 GB RAM (more recommended)
+* min. 100 GB free disk space for local installation 
+  - (min/max similar to Docker?)
+* Linux OS (we recommend Ubuntu 18.04 for native installation)
+* Python 3.5+ (we recommend 3.6 or 3.7;* **don't use 3.8, as there are several packages unavailable for it***)
```

Comment:
We could be more precise here: there are no pre-build Python packages for Tensorflow and other GPU related software.

Reply:
It is possible to run `make all -k` with Python 3.8, and it will build enough working processors for some OCR-D workflows.

Reply:
At the moment, we just cannot in good faith support 3.8 even though most processors have no issues. c.f. the discussion with @Shanksum in the lobby or @cneud's experiments with ubuntu 20.04. It is best to install/enable a supported python version for new users at the moment.

Reply:
People read and remember such information, and they might follow that rule even if it is no longer true. Adding "currently" and some date information would clearly show that this will change in the future.

As there are more and more recent Linux distros with Python 3.8 I expect that it won't take long until Tensorflow packages are available.

Maybe we should add a short test to ocrd_all which checks the known problem with Python 3.8 and Tensorflow and skips all affected packages when needed, telling users why it did so. That would have helped in both cases.

Reply:
> People read and remember such information, and they might follow that rule even if it is no longer true. Adding "currently" and some date information would clearly show that this will change in the future.

> As there are more and more recent Linux distros with Python 3.8 I expect that it won't take long until Tensorflow packages are available.

I've tried to clarify that and added a provisionary plan to fully support Python 3.8 in Q4 2020 or Q1 20201

> Maybe we should add a short test to ocrd_all which checks the known problem with Python 3.8 and Tensorflow and skips all affected packages when needed, telling users why it did so. That would have helped in both cases.

> A similar test would also help users on non-Intel platforms.

We should go through the processors (at least those that have CI) and enable testing against Python 3.8 to see which fail (and to make it easier to confidently support 3.8 asap).
 
I am not quite sure how such a test would look like in ocrd_all, but I am intrigued, also about the non-Intel platforms - can we continue discussion in an ocrd_all issue/PR?